app/src/main/java/com/example/qrrush/MainActivity.java

```diff
@@ -70,8 +71,35 @@ protected void onCreate(Bundle savedInstanceState) {
         Log.d("TAG", UserUtil.getUsername(MainActivity.this));
         String username = UserUtil.getUsername(getApplicationContext());
 
+        /**
+         * Test data for QR codes (Testing the sorting system) don't remove!
+         * TODO remove once users can scan QR codes into fire base
+         * ================================================
+         */
         ArrayList<QRCode> qrCodes = new ArrayList<>();
-        // Get everything from firebase
+        byte[] b = new byte[20];
```

Comment:
Make theses sorting cases a test. In the actual app we only want to get things from firebase.

Reply:
note that this is already a PR (#71), so once its merged just delete these.